backend/api/tests/test_admin.py

```
@@ -201,4 +205,175 @@ def test_reset_access_count_action(self):
         
         # Verify the access count was reset
         self.video_share.refresh_from_db()
-        self.assertEqual(self.video_share.access_count, 0)
\ No newline at end of file
+        self.assertEqual(self.video_share.access_count, 0)
+
+
+class VideoAdminTest(AdminTestCase):
+    def create_request_with_messages(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_request_with_messages`

(ANN201)